integration-tests/src/test/java/com/redhat/service/bridge/integration/tests/common/Utils.java

```diff
@@ -0,0 +1,11 @@
+package com.redhat.service.bridge.integration.tests.common;
+
+public class Utils {
+
+    public static String getSystemProperty(String parameters) {
```

Comment:
Is it needed to keep the method in separate class? It seems to be used just in `BridgeUtils.java`

Reply:
I see, there will be more functionality added in next PRs.

Reply:
It was just in foresee of the next changes ... I will set it back to `BridgeUtils`